setup.py

```diff
@@ -34,6 +34,9 @@
 req_lines = [line.strip() for line in open("requirements.txt").readlines()]
 install_reqs = list(filter(None, req_lines))
 
+test_req_lines = [line.strip() for line in open("test-requirements.txt").readlines()]
```

Comment:
and here you're only reading test-requirements.txt, not test-requirements-py3.txt !